docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  monero:
+    container_name: monero
+    build:
+      context: .
+    restart: always
+    ports:
+      - 18080:18080
+      - 18081:18081
+      - 18082:18082
+      - 18083:18083
+      - 18084:18084
+    command:
+      - --p2p-bind-ip=0.0.0.0
+      - --p2p-bind-port=18080
+      - --rpc-bind-ip=0.0.0.0
+      - --rpc-bind-port=18081
+      - --non-interactive
+      - --rpc-ssl=disabled
+      - --disable-rpc-ban
+      - --confirm-external-bind
+      - --zmq-pub=tcp://0.0.0.0:18084
+      - --zmq-rpc-bind-port=1882
+      - --zmq-rpc-bind-ip=0.0.0.0
+      - --log-level=4
```

Comment:
Why `--rpc-ssl=disabled`? Also `--log-level=4` is too high, it will cause performance issues due to massive amount of logs written.

Reply:
It's just a template for development and everyone could modify them by their needs. I think Server-side services communicate under the Docker network by service name in most cases and SSL isn't required between them.
However, it seems it's better to change them let me remove these and add some comments from "monerod --help" to it.